web/src/javascript/network.js

```diff
@@ -38,8 +38,11 @@ export function sendAdminRequest(url, method, data = {}) {
     body: JSON.stringify(data),
   }).then(response => response.json())
     .then(response => {
-      if(response.data !== "ok")
+      if(response.data !== "ok"){
+        if(response.error === "forbidden")
```

Comment:
I think it is a bit flaky to look at the error message. Would there be a clean way to look at the status code before converting to `.json()`? What about:

```js
}).then(response => {
  if (response.status === 403) {
    window.location //...
  } else {
    response.json().then(response => {
      if (!response.data) throw response.error;
      else return response.data;
    });
  }
});
```

(making them nested here after reading [this SO answer](http://stackoverflow.com/a/20715224/395386))